content/en/docs/instrumentation/java/getting-started.md

```diff
@@ -1,138 +1,216 @@
-    to the JAR file.{{% /alert %}}
-2.  Set and export variables that specify the Java agent JAR and a [console
-    exporter][], using a notation suitable for your shell/terminal environment
-    &mdash; we illustrate a notation for bash-like shells:
-
-    ```console
-    $ export JAVA_OPTS="-javaagent:PATH/TO/opentelemetry-javaagent.jar"
-    $ export OTEL_TRACES_EXPORTER=logging
-    $ export OTEL_METRICS_EXPORTER=logging
-    ```
-
-    {{% alert title="Important" color="warning" %}}Replace `PATH/TO` above, with
-    your path to the JAR.{{% /alert %}}
-
-3.  Run the **server** as a background process. For example, for bash-like
-    shells run:
-
-    ```console
-    $ ./build/install/examples/bin/hello-world-server &
-    [otel.javaagent 2022-03-19 13:38:16:712 +0000] [main] INFO io.opentelemetry.javaagent.tooling.VersionLogger - opentelemetry-javaagent - version: 1.12.0
-    Mar 19, 2022 1:38:18 PM io.grpc.examples.helloworld.HelloWorldServer start
-    ...
-    ```
-
-    Note the output from the `otel.javaagent`.
-
-4.  From the _same_ terminal, run the **client**:
-
-    ```console
-    $ ./build/install/examples/bin/hello-world-client
-    [otel.javaagent 2022-03-19 13:38:48:462 +0000] [main] INFO io.opentelemetry.javaagent.tooling.VersionLogger - opentelemetry-javaagent - version: 1.12.0
-    Mar 19, 2022 1:38:50 PM io.grpc.examples.helloworld.HelloWorldClient greet
-    INFO: Will try to greet world ...
-    [Trace output elided -- see below for details]
-    Mar 19, 2022 1:38:51 PM io.grpc.examples.helloworld.HelloWorldClient greet
-    INFO: Greeting: Hello world
-    ```
-
-5.  Stop the server process. For example, for bash-like shells run:
-
-    ```console
-    $ jobs
-    [1]+  Running                 ./build/install/examples/bin/hello-world-server &
-    $ kill %1
-    ...
-    *** server shut down
-    ```
-
-At step 4, you should have seen trace output from the server and client that
-looks something like this (trace output is line-wrapped for convenience):
+application at launch time. While you can [configure the Java agent][] in a
+number of ways, the steps below use environment variables.
+
+1. Download [opentelemetry-javaagent.jar][] from [Releases][] of the
+   `opentelemetry-java-instrumentation` repo. The JAR file contains the agent
+   and all automatic instrumentation packages:
+
+   ```console
+   curl -L -O https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/latest/download/opentelemetry-javaagent.jar
+   ```
+
+   {{% alert color="info" %}}<i class="fas fa-edit"></i> Take note of the path
+   to the JAR file.{{% /alert %}}
+
+2. Set and export variables that specify the Java agent JAR and a [console
+   exporter][], using a notation suitable for your shell/terminal environment
+   &mdash; we illustrate a notation for bash-like shells:
+
+   ```console
+   $ export JAVA_OPTS="-javaagent:PATH/TO/opentelemetry-javaagent.jar" \
```

Comment:
AFAIK `JAVA_TOOL_OPTIONS` is the universally recognized env var name -- at least that what's been working for me (temurin 17)
```suggestion
   $ export JAVA_TOOL_OPTIONS="-javaagent:PATH/TO/opentelemetry-javaagent.jar" \
```

Reply:
Good call, I ran into that issue myself but forgot to fix that!